### DISCUSSION TITLE
Deactivate focus-scope on a dialog

### DISCUSSION THREAD

**Discussion by flipvh** [radix-ui/primitives#2533] [Help] [answered]
Hi there! We are now using Shadcn-UI which is itself using Radix-UI. I am using a Sheet component, based on the Radix Dialog. As we also have a UI mode where the sheet remains open, we would like to interact with the rest of the page. But for this, we need to be able to disable the focus trap / focus scope. 

Are there others with the same situation, where you want to be able to disable the focus trap? 

Because it looks like focus trap is on by default (which makes perfect sense for a Dialog usage pattern) and there is no option to disable it? Or am I overlooking it? Thanks!

Here a screenshot of the Sheet next to the content, which I would like to interact with (without a focus trap on the sheet (Dialog)).

<img width="1446" alt="Screenshot 2023-11-18 at 10 17 59" src="https://github.com/radix-ui/primitives/assets/3855558/a64f35f7-abe2-49ad-9d77-367300055642">

**Comment by flipvh** [comment]
Its perhaps a bit related to this one: https://github.com/radix-ui/primitives/discussions/2452

**Comment by joaom00** [comment, marked as answer]
You can use `modal` prop on `Root`

> The modality of the dialog. When set to true, interaction with outside elements will be disabled and only dialog content will be visible to screen readers.

**Reply by andresiacobelli** [reply]
Isnt there anyway of avoiding it for a specific component??
